lib/utils/token/getEntitlements.ts

```diff
@@ -0,0 +1,38 @@
+import { BaseAccountResponse } from "../../types";
+import { callAccountApi } from "./accountApi/callAccountApi";
+
+/**
+ * Fetches entitlements from the account API.
+ * @returns {Promise<{ name: keyof T; value: V } | EntitlementsResponse>}
+ * @template T - Type of the decoded JWT.
+ * @template V - Type of the entitlement value.
+ */
+export const getEntitlements = async (): Promise<EntitlementsResponse> => {
+  return callAccountApi<EntitlementsResponse>("account_api/v1/entitlements");
+};
+
+type Entitlement = {
+  id: string;
+  fixed_charge: number;
+  price_name: string;
+  unit_amount: number;
+  feature_code: string;
```

Comment:
@DanielRivers I think I made this `feature_key` in the end as we use `key` in the product